libs/blockprotocol/package.json

```diff
@@ -45,6 +45,7 @@
     "@local/tsconfig": "0.0.0-private",
     "@types/command-line-args": "^5.2.0",
     "@types/command-line-usage": "^5.0.2",
+    "@types/node": "^18.11.18",
```

Comment:
I'm not sure why this occurred in this PR, but to get lints to pass I had to add `@types/node` to a few packages and set `skipLibCheck: true` in some of the TS configs.

cc @kachkaev

Reply:
How about the added `skipLibCheck: true`s?

Reply:
Was also able to remove them: https://github.com/blockprotocol/blockprotocol/pull/888/commits/e56edb9e4ddedc13ae717dd086c22cbb4e182d0d